markdown/wiki/faq/android/RENDERERS.md

```diff
@@ -1,4 +1,4 @@
-# Renderers
+# Renderers <!-- TODO: Fix up the descriptions here (Is holy still the main renderer??) and presumably remove mention of LTW? -->
```

Comment:
Remove LTW, it is not supported anymore.

AFAIK GL4ES is indeed still the default.